src/consensus/pbft/libbyz/Replica.cpp

```diff
@@ -2174,6 +2180,7 @@ std::unique_ptr<ExecCommandMsg> Replica::execute_tentative_request(
     last_tentative_execute,
     max_local_commit_value,
     stash_replier,
+    request.user_id(),
```

Comment:
`user_id` is the thread id right? Should we rename it to `thread_id()` to avoid any future confusion?